### PR TITLE
yaourt: add page

### PR DESCRIPTION
**Pull request by dylanrees**

### CODE REVIEW

File: pages/linux/yaourt.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+- install a new package
+
+`yaourt -S package-name`
+
+- remove a package and its dependencies
+
+`yaourt -Rs package-name`
+
+- search the package database for a keyword
```
Comment (felixonmars) [inline pages/linux/yaourt.md]: You might want to mention AUR here too, since you did that for the first example :)

**Comment by dylanrees** [comment]
How's this?

File: pages/linux/yaourt.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+`yaourt -Rs package-name`
+
+- search the package database for a keyword (including AUR)
+
+`yaourt -Ss package-name`
+
+- list installed packages, versions, and repositories (AUR packages will be listed under the repository name 'local')
+
+`pacman -Q`
```
Comment (igorshubovych) [inline pages/linux/yaourt.md]: There is a separate page for `pacman`. IMO it should not be here.

Comment (waldyrious) [inline pages/linux/yaourt.md]: Perhaps mention it in the main description.

**Comment by dylanrees** [comment]
Oops, that was a mistake.  Corrected.

**Comment by igorshubovych** [comment]
Could you plz squash the commits?

**Comment by dylanrees** [comment]
Sorry, what do you mean by that?

**Comment by igorshubovych** [comment]
@dylanrees 
https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md#submitting-a-pull-request, item 9

You have 3 commits in this pull request. And we do not want to

**Pull request closed without merging**